ansible/molecule/ec2_full/molecule.yml

```diff
@@ -39,6 +39,20 @@ platforms:
     region: us-east-1
     instance_type: t3.small
     ssh_user: ec2-user
+  - name: konvoyimage-rhel7.9-${USER:-ci}-${HOSTNAME:-local}
+    image_search_name: "RHEL-7.9_HVM-*"
+    image_search_owner: "309956199498"
+    region: us-east-1
+    instance_type: p2.xlarge
```

Comment:
Why do we need GPU nodes for these tests? IIUC then t3.small should do, like above for centos.

Reply:
oops, nice catch